src/NUnitFramework/tests/AsyncExecutionApiAdapter.Fixture-based.FSharp.cs

```diff
@@ -0,0 +1,90 @@
+// ***********************************************************************
+// Copyright (c) 2018 Charlie Poole, Rob Prouse
```

Comment:
I wrote the code in 2018, so what I did in another file was 2018–2019. Is that how it works, or should the date be when it was contributed rather than when it was created?

Reply:
Ah, just saw your other suggestion to remove the '2018–' part. Sounds good.

Reply:
Actually, I'm unsure about the case when the code is written in one year, but contributed in another year.

Reply:
Some open-source projects don't put copyright years in the headers. I keep thinking longingly about that.

Reply:
I have often moved things from an older project, e.g. nunitlite to nunit. I just use the most inclusive date range.